spec/system/banners/new_spec.rb

```diff
@@ -49,14 +49,32 @@
     within "#banners" do
       click_on "Edit", match: :first
     end
-    fill_in "banner_expires_at", with: 7.days.ago.strftime("%m%d%Y\t%I%M%P")
+    fill_in "banner_expires_at", with: 2.days.from_now.strftime("%m%d%Y\t%I%M%P")
     click_on "Submit"
 
     visit banners_path
     expect(page).to have_text("Expiring Announcement")
 
     visit root_path
-    expect(page).not_to have_text("Please fill out this survey.")
+    expect(page).to have_text("Please fill out this survey.")
+  end
+
+  it "does not allow creation of banner with an expiration time set in the past" do
+    sign_in admin
+
+    freeze_time do
+      current_time = Time.current
+      visit banners_path
+      page.driver.browser.manage.add_cookie(name: "browser_time_zone", value: "America/Chicago")
+      click_on "New Banner"
+      fill_in "Name", with: "Announcement"
+      fill_in "banner_expires_at", with: (current_time - 1.hour).strftime("%m%d%Y\t%I%M%P")
+      fill_in_rich_text_area "banner_content", with: "Please fill out this survey."
+      click_on "Submit"
+
+      message = page.find("#banner_expires_at").native.attribute("validationMessage")
+      expect(message).to eq("Value must be #{current_time.in_time_zone("America/Chicago").strftime("%m/%d/%Y, %I:%M %p or later.")}")
+    end
```

Comment:
I know that I suggested using `page.driver.browser.manage.add_cookie(name: "browser_time_zone", value: "America/Chicago")` but it doesn't seem to work as expected. It feels more tricky that its worth to test these error messages in system tests. I think we should simplify this test to just check that an expired banner cannot be created but not actually check the error.

I do think that checking the cookie logic is important and I think we should move those tests to requests specs. Off the top of my head the tests that matter for edge cases are:
- client in ahead of server, sets expired at time that is behind current time for client but ahead for server => this should be invalid
- client is behind of server, set expired at time that is ahead of current time for client but behind for server => this should be valid

Also when checking the error values it is much easier to read if they are in a form like 
```
expect(something).to have_text("Invalid time: must be after 1/1/1 12:00 PM")
``` 
instead of 
```
"Value must be #{current_time.in_time_zone("America/Chicago").strftime("%m/%d/%Y, %I:%M %p or later.")}"
```
That should be simple to do by just traveling to a specific date instead of one relative to today.